src/sas/qtgui/Perspectives/SizeDistribution/SizeDistributionPerspective.py

```diff
@@ -792,8 +792,10 @@ def updateStatistics(self, result):
                 )
             else:
                 converge_msg = f"Full fit converged after on average {np.mean(result.num_iters):.1f} iterations"
+            self.lblConvergence.setStyleSheet("color: black;")
         else:
             converge_msg = "Not converged"
```

Comment:
Am thinking @ldschmitt might like to see more cautionary note? Maybe adding to the string? Something like:
"Not converged! Proceed with extreme caution - Try increasing the weight factor"
or maybe just
"Not converged!  Try increasing the weight factor"

or as is works too